docs/contributing/pull-request-guidelines/index.md

```diff
@@ -3,79 +3,133 @@
-     - Generally, the author is expected to merge it to feel responsible for their pull request.
+     - The author is encouraged to merge the pull request to feel responsible for their own pull request.
      - If the author does not have write access, ask the reviewers or maintainers.
-     - It is the author's responsibility to care about their own pull request until it is merged.
 
 ## Pull request rules
 
-### Follow `Conventional Commits` (required, automated)
+### Use an appropriate pull request template (required, non-automated)
 
 #### Rationale
 
-- It can generate categorized changelog, for example using [git-cliff](https://github.com/orhun/git-cliff).
+- The unified style of descriptions by templates can make reviews efficient.
+
+#### Example
+
+1. Select the appropriate template, as shown in [this video](https://user-images.githubusercontent.com/31987104/184344710-2adee239-799f-4fdf-bfab-be76345bfac1.mp4).
+2. Read the selected template carefully and fill the required content.
+3. Check the checkboxes during a review.
+   - There are [pre-review checklist](https://github.com/autowarefoundation/autoware/blob/44c70d33825617b56d8de5e6fe921000238238bd/.github/PULL_REQUEST_TEMPLATE/standard-change.md#pre-review-checklist-for-the-pr-author) and [post-review checklist](https://github.com/autowarefoundation/autoware/blob/44c70d33825617b56d8de5e6fe921000238238bd/.github/PULL_REQUEST_TEMPLATE/standard-change.md#post-review-checklist-for-the-pr-author) for the author.
+
+### Set appropriate reviewers after creating a pull request (required, partially automated)
+
+#### Rationale
+
+- Pull requests must be reviewed by appropriate reviewers to keep the quality of the codebase.
+
+#### Example
+
+- For most ROS packages, reviewers will be automatically assigned based on the `maintainer` information in `package.xml`.
+- If no reviewer is assigned automatically, assign reviewers manually following the instructions in [GitHub Docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review).
+  - You can find the reviewers by seeing the `.github/CODEOWNERS` file of the repository.
+- If you are not sure the appropriate reviewers, ask `@autoware-maintainers`.
+- If you have no rights to assign reviewers, mention reviewers instead.
+
+### Apply Conventional Commits to the pull request title (required, automated)
+
+#### Rationale
+
+- [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) can generate categorized changelogs, for example using [git-cliff](https://github.com/orhun/git-cliff).
 
 #### Example
 
 ```text
 feat(trajectory_follower): add an awesome feature
 ```
 
-Note that you have to start the description part (`add an awesome feature`) with a lowercase.
+!!! note
+
+    You have to start the description part (here `add an awesome feature`) with a lowercase.
```

Comment:
Copied from https://github.com/autowarefoundation/autoware-documentation/pull/193#discussion_r962951572.

except -> except for?

```suggestion
    The commit message should be all lowercase except specific abbreviations.
```

Reply:
Actually, since there is an optional body and it starts with an upper case, I think it's better to mention only the description here.
@xmfcx What do you think?
![image](https://user-images.githubusercontent.com/31987104/188528722-3dcfc94f-925f-474d-999f-ac71da880ecb.png)
![image](https://user-images.githubusercontent.com/31987104/188528801-16b58564-60b9-4b0c-8b69-526ea2e14897.png)

Reply:
> Copied from [#193 (comment)](https://github.com/autowarefoundation/autoware-documentation/pull/193#discussion_r962951572).
> 
> except -> except for?

Yes, "except for" would be correct.

Reply:
Also I knew it wrong.

I thought that:

`git commit -m "this is the message" -m "this is the description"`

But correct way is:

`git commit -m "this is the subject" -m "this is the body"`

So we can keep it as it is, sorry for the confusion.